capture-sdk/sdk/src/main/java/net/gini/android/capture/help/view/HelpNavigationBarBottomAdapter.kt

```diff
@@ -7,11 +7,16 @@ import net.gini.android.capture.databinding.GcHelpNavigationBarBottomBinding
 import net.gini.android.capture.view.InjectedViewAdapter
 
 /**
- * Implement this interface if you would like custom action triggered when tapping back navigation
+ * Implement this interface to add back navigation to a custom view on bottom navigation bar
  * and pass it to the {@link GiniCapture.Builder#setHelpNavigationBarBottomAdapter(HelpNavigationBarBottomAdapter)}
  */
 interface HelpNavigationBarBottomAdapter : InjectedViewAdapter {
 
+    /**
+     * <p>
+     *  Set a listener for back navigation on a view on bottom navigation bar layout.
```

Comment:
[Suggestion] I would suggest to only say here `Set a listener for the back navigation button.` 

I don't think we have to repeat `bottom navigation bar layout`. I find it's implied by the context. But I have no strong oppionin. Feel free to keep it. It's better to have more information than less.